apps/studio/components/interfaces/BranchManagement/BranchSelector.tsx

```diff
@@ -53,7 +53,7 @@ export const BranchSelector = ({
           type={type}
           disabled={disabled || availableBranches.length === 0 || isUpdating}
         >
-          {isUpdating ? 'Creating...' : 'Create merge request'}
+          {isUpdating ? 'Creating...' : 'New merge request'}
```

Comment:
it looked like this before
<img width="1103" height="228" alt="image" src="https://github.com/user-attachments/assets/f4939460-d327-4e29-a1e8-a8d5c7d94c1d" />

which the "Merge request" button didn't feel like a CTA, so changing the copy there and here as well, so it looks like this, in hopes to indicate that these 2 buttons do different things

<img width="1085" height="226" alt="image" src="https://github.com/user-attachments/assets/63bf4664-34ea-4c48-93fd-5173752cfcd9" />